### PR TITLE
timoni: init at 0.10.0

### PR DESCRIPTION
**Pull request by janvotava**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by stefanprodan — APPROVED** [review]
Thank you @janvotava 🥇

**Comment by stefanprodan** [comment]
@janvotava do we set the version when we build the binary? We do this upstream like so https://github.com/stefanprodan/timoni/blob/main/.goreleaser.yml#L9

**Comment by janvotava** [comment]
> @janvotava do we set the version when we build the binary? We do this upstream like so https://github.com/stefanprodan/timoni/blob/main/.goreleaser.yml#L9

Nope, I will fix that.

```
timoni version 0.0.0-dev.0

File: pkgs/applications/networking/cluster/timoni/default.nix [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    --bash <($out/bin/timoni completion bash) \
+    --fish <($out/bin/timoni completion fish) \
+    --zsh <($out/bin/timoni completion zsh)
+  '';
+
+  meta = with lib; {
+    homepage = "https://timoni.sh";
+    changelog = "https://github.com/stefanprodan/timoni/releases/tag/${src.rev}";
+    description = "Timoni is a package manager for Kubernetes, powered by CUE and inspired by Helm.";
+    platforms = platforms.linux ++ platforms.darwin;
```
Comment (emilylange) [inline pkgs/applications/networking/cluster/timoni/default.nix:43]:
```suggestion
```

That's already the default in `buildGoModule`; redundant

File: pkgs/applications/networking/cluster/timoni/default.nix [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    installShellCompletion --cmd timoni \
+    --bash <($out/bin/timoni completion bash) \
+    --fish <($out/bin/timoni completion fish) \
+    --zsh <($out/bin/timoni completion zsh)
+  '';
+
+  meta = with lib; {
+    homepage = "https://timoni.sh";
+    changelog = "https://github.com/stefanprodan/timoni/releases/tag/${src.rev}";
+    description = "Timoni is a package manager for Kubernetes, powered by CUE and inspired by Helm.";
```
Comment (emilylange) [inline pkgs/applications/networking/cluster/timoni/default.nix:42]:
```suggestion
    description = "A package manager for Kubernetes, powered by CUE and inspired by Helm";
```

See https://nixos.org/manual/nixpkgs/unstable/#var-meta-description

**Review by emilylange — CHANGES_REQUESTED** [review]
Please also squash/rebase
`timoni: refactor - tweak ldflags` into 
`timoni: init at 0.10.0`.

There should only be two commits in total:
1. `maintainers: add votava`
1. `timoni: init at 0.10.0`

Thank you.

**Comment by janvotava** [comment]
@emilylange Thanks for the feedback. Everything should be fixed.

**Pull request merged**